lib/ansible/modules/cloud/ovirt/ovirt_role.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: ovirt_role
+short_description: Module to manage roles in oVirt/RHV
+version_added: "2.8"
+author: "Martin Necas (@mnecas)"
+description:
+    - "Module to manage roles in oVirt/RHV."
+options:
+    name:
+        description:
+            - "Name of the role to manage."
+    id:
+        description:
+            - "ID of the role to manage."
+    state:
+        description:
+            - "Should the role be present/absent."
+        choices: ['present', 'absent']
+        default: present
+    mutable:
```

Comment:
I think you can't create imutable role, and all roles created by users are mutable, so this parameter is not needed.